packages/synapse-react-client/src/components/row_renderers/utils/CardFooter.tsx

```diff
@@ -95,40 +96,34 @@ class CardFooter extends Component<CardFooterProps, State> {
 
     return value
   }
-  renderRows = (values: string[][], limit: number, isDesktop: boolean) => {
-    return values.map((kv, index) => {
+  renderRows = (values: CardLabel[], limit: number, isDesktop: boolean) => {
```

Comment:
Converted the string array to a `CardLabel` object. Just makes it easier to read/reason about